Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/ThemeDefinition.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// The main settings found in Themes
```

Comment:
Hm...what do you mean by "main settings in Themes"? Can you maybe provide more details?